app/connectors/httpParsers/CustomerDetailsHttpParser.scala

```diff
@@ -21,35 +21,36 @@ import models.CustomerDetails
 import models.errors._
 import play.api.http.Status._
 import uk.gov.hmrc.http.{HttpReads, HttpResponse}
-import utils.LoggerUtil
+import utils.LoggingUtil
 
-object CustomerDetailsHttpParser extends LoggerUtil {
+object CustomerDetailsHttpParser extends LoggingUtil {
 
   implicit object CustomerDetailsReads extends HttpReads[HttpResult[CustomerDetails]] {
 
     override def read(method: String, url: String, response: HttpResponse): HttpResult[CustomerDetails] = {
+      implicit val res: HttpResponse = response
 
       response.status match {
         case OK =>
-          logger.debug("[CustomerCircumstancesHttpParser][read]: Status OK")
+          debug("[CustomerCircumstancesHttpParser][read]: Status OK")
           response.json.validate[CustomerDetails].fold(
             invalid => {
-              logger.debug(s"[CustomerCircumstancesHttpParser][read]: Invalid Json - $invalid")
```

Comment:
just check invalid isnt returning any PII please

Reply:
this does not contain any PII so it should bee fine